app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -213,33 +213,28 @@ def remove_subscriptions
 
     api :PUT, "/activation_keys/:id/content_override", N_("Override content for activation_key")
     param :id, :number, :desc => N_("ID of the activation key"), :required => true
-    param :content_override, Hash, :desc => N_("Content override parameters"), :deprecated => true do
+    param :content_override, Hash, :desc => N_("Content override parameters") do
       param :content_label, String, :desc => N_("Label of the content"), :required => true
       param :value, String, :desc => N_("Override to a boolean value or 'default'"), :required => true
     end
-    param :content_overrides, Array, :desc => N_("Array of Content override parameters") do
+    param :content_overrides, Array, :desc => N_("Array of Content override parameters to be added in bulk") do
       param :content_label, String, :desc => N_("Label of the content"), :required => true
       param :value, String, :desc => N_("Override value. Provide a boolean value if name is 'enabled'"), :required => false
       param :name, String, :desc => N_("Override parameter key or name. Note if name is not provided the default name will be 'enabled'"), :required => false
       param :remove, :bool, :desc => N_("Set true to remove an override and reset it to 'default'"), :required => false
     end
     def content_override
       content_overrides = []
-      if params[:content_override]
-        ::Foreman::Deprecation.api_deprecation_warning("The parameter content_override will be removed in Katello 3.5. Please update to use the content_overrides parameter.")
-        content_override_params = {:content_label => params[:content_override][:content_label]}
-        value = params[:content_override][:value]
-        if value == "default"
-          content_override_params[:remove] = true
-        elsif value
-          content_override_params[:value] = ::Foreman::Cast.to_bool(value)
-        end
-        content_overrides << content_override_params
-      elsif params[:content_overrides]
-        content_overrides = params[:content_overrides]
-      end
+      content_overrides += [params[:content_override]] if params[:content_override]
```

Comment:
Are we no longer deprecating this param?

Reply:
I see - you opted not to. Couldn't we go ahead and do it?

Reply:
I don't see a huge reason to, I mentioned that in my commit message:
> The 'content_override' param was deprecated previously, but not actually removed (this is a whole other
issue with how we handle deprecations). This param would just be used for the 'enabled' parameter and
was hardcoded to update only that. I don't see a huge reason to remove it, so I removed the deprecation warning.
and I updated to code to pass it as a bulk param so it uses the same validations as the bulk parameters.

Reply:
I'm not sure if we should, but maybe we can discuss further. 

My view is it works fine as is and I think with some clarification in the API docs that the single param is hardcoded to change "enabled", it could stay useful and not conflict with anything else. This to me doesn't feel like a parameter changed, but additional functionality to add bulk params was added so I don't see a huge harm in keeping it around and not deprecating.

But open to other opinions from the team!

Reply:
Thinking about this more....

I could also see the `content_override` parameter being actually removed as it looks like this is the only place the API supports this and I see `content_overrides` is the pattern everywhere else. I was worried about the UI and hammer, but it looks like those have been updated to use `content_overrides` (though we should confirm these workflows)

After all, users have had since 3.5 to update :laughing: 

I can see an argument for either way, its nice to not break APIs when possible, but also we have a new pattern and have propery deprecated... 

Thats enough thinking out loud for me... @jturel curious to hear your thoughts :wink:

Reply:
I'm all for removing it since the deprecation warning has been there for ~2.5 years. Since you mentioned that hammer is using the new param I'm especially convinced. I'd wager the set of users automating this through the API and still using the old parameter are small. Do it!

Reply:
If you do remove my only request is to make sure any test coverage that is lost is updated to use the new param (if there are no tests for a particular behavior w. that new param already)